### PR TITLE
MGMT-1722 Modify 2nd ignition to create parameter file to Openshift for DHCP address allocation

### PR DESCRIPTION
**Pull request by ori-amizur**

### CODE REVIEW

File: render_files.py [hunk 1]
```diff
@@ -41,6 +41,22 @@ def upload_to_aws(s3_client, local_file, bucket, s3_file):
         return False
 
 
+def add_dhcp_allocation_file(ignition_file, dhcp_allocation_file):
+    try:
+        with open(ignition_file, "r") as file_obj:
+            data = json.load(file_obj)
+            storage_files = data['storage']['files']
+            entry = {"filesystem": "root",
```
Comment (yevgeny-shnaidman) [inline render_files.py:49]: Please , add a TODO note here to change it once we move to ignition 3.0

**Review by yevgeny-shnaidman — APPROVED** [review]

File: render_files.py [hunk 2]
```diff
@@ -195,8 +212,14 @@ def main():
...
 
+    ignition_file = "%s/bootstrap.ign" % config_dir
+
     # update BMH configuration in boostrap ignition
-    update_bmh_files("%s/bootstrap.ign" % config_dir, cluster_id, inventory_endpoint, openshift_token(config_dir))
+    update_bmh_files(ignition_file, cluster_id, inventory_endpoint, openshift_token(config_dir))
+
+    if dhcp_allocation_file:
+        # Add dhcp allocation file if needed to ignition
+        add_dhcp_allocation_file(ignition_file, dhcp_allocation_file)
```
Comment (yevgeny-shnaidman) [inline render_files.py:222]: Maybe you can also add a test ( the same way as for BMH), that will allow us testing and verifying ignition locally (on laptop)

**Review by romfreiman — CHANGES_REQUESTED** [review]
This is not the way to go. We should update the MCS not the first igntion

**Comment by romfreiman** [comment]
Please dont merge. This is incorrect solution

**Review by romfreiman — CHANGES_REQUESTED** [review]
Please dont merge. this is incorrect solution

**Review by eranco74 — APPROVED** [review]

File: render_files.py (hunk 2 above)
Comment (eranco74) [inline render_files.py:222]: You need to update the master ignition not the bootstrap, no?

**Review by eranco74 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**